go.sum

```diff
@@ -10,6 +10,8 @@ github.com/alexflint/go-filemutex v0.0.0-20171022225611-72bdc8eae2ae/go.mod h1:C
 github.com/buger/jsonparser v0.0.0-20180808090653-f4dd9f5a6b44/go.mod h1:bbYlZJ7hK1yFx9hf58LP0zeX7UjIGs20ufpu3evjr+s=
 github.com/containernetworking/cni v0.7.1 h1:fE3r16wpSEyaqY4Z4oFrLMmIGfBYIKpPrHK31EJ9FzE=
```

Comment:
@zshi-redhat did you run go mod tidy as well ?

Reply:
Yes, I ran `go mod tidy` with golang `go version go1.15.6 linux/amd64`.